packages/openjdk17.rb

```diff
@@ -0,0 +1,43 @@
+require 'package'
+
+class Openjdk17 < Package
+  description 'The JDK is a development environment for building applications, applets, and components using the Java programming language.'
+  homepage 'https://openjdk.org/'
+  version '17.36.19'
+  license 'GPL-2'
+  compatibility 'all'
+  source_url({
+    aarch64: 'https://cdn.azul.com/zulu/bin/zulu17.36.17-ca-jdk17.0.4.1-linux_aarch64.tar.gz',
+     armv7l: 'https://cdn.azul.com/zulu-embedded/bin/zulu17.36.19-ca-jdk17.0.4.1-linux_aarch32hf.tar.gz',
```

Comment:
We'll need to use the aarch32hf binary for aarch64 too since ChromeOS still implements a 32-bit userland.